carina-webdriver/src/main/java/com/qaprosoft/carina/core/gui/mobile/devices/android/phone/pages/fakegps/FakeGpsPage.java

```diff
@@ -109,7 +110,7 @@ public boolean locationSearch(String location) {
                 inputLocationNew.type(location);
                 ((AndroidDriver<AndroidElement>) getDriver()).pressKeyCode(AndroidKeyCode.ENTER);
                 ((AndroidDriver<AndroidElement>) getDriver()).pressKeyCode(AndroidKeyCode.KEYCODE_SEARCH);
-                pause(3);
+                CommonUtils.pause(3);
```

Comment:
@vdelendik Should we use CommmonUtils.pause(3) here or inherited method from DriverHelper.class?